src/targets/ttn/analog/downlink/template.js

```diff
@@ -0,0 +1,44 @@
+// these options can be edited
+// available hardware types can be found here:
+// https://github.com/jooby-dev/jooby-docs/blob/main/docs/analog/basics.md#hardware-types
+var config = {
+    // required field
+    hardwareType: 0,
+};
+
+
+//#region [autogenerated jooby-codec bundle]
+/*{{bundle}}*/
+//#endregion
+
+
+function encodeDownlink ( input ) {
+    // input has the following structure:
```

Comment:
how about move it before function and make similar to https://github.com/jooby-dev/jooby-codec-adapters/blob/main/src/targets/chirpstack4/analog/template.js#L12 ?

Reply:
done